cosmos-sdk-proto/src/type_urls.rs

```diff
@@ -9,6 +9,83 @@ use crate::{cosmos, ibc, traits::TypeUrl};
 #[cfg(feature = "cosmwasm")]
 use crate::cosmwasm;
 
+#[cfg(feature = "cosmwasm")]
```

Comment:
I think it would make sense to logically group these with the other `cosmwasm`-gated impls.

It might even be worth considering a submodule

Reply:
yes, also good would be for the proto-build to auto generate the types, in the worst case via text parsing that should not be that hard, anyways I manually went over all the current governance proposal types and for now I have everything I need.